app/views/sessions/log_form.html.erb

```diff
@@ -0,0 +1,7 @@
+<h2>Log In</h2>
+
+<%= form_tag do %>
+<%= label_tag(:username, "Username") %>
+<%= text_field_tag(:username) %>
```

Comment:
logging in doesn't work as is-- if you change this form to use the symbol `:name` instead of `:username` it does!

The symbol you specify here (in this case, `:username`) gets stored into the params hash. This params hash is what is used while creating a user (see `sessions_controller.rb` in line 10). I'm adding a comment at that point too